src/histolab/tiler.py

```diff
@@ -182,13 +182,29 @@ def extract(self, slide: Slide):
         ----------
         slide : Slide
             Slide from which to extract the tiles
+
+        Raises
+        ------
+        TileSizeError
+            If the tile size is larger than the slide size
+        LevelError
+            If the level is not available for the slide
         """
         if self.level not in slide.levels:
             raise LevelError(
                 f"Level {self.level} not available. Number of available levels: "
                 f"{len(slide.levels)}"
             )
 
+        if (
+            self.tile_size[0] > slide.level_dimensions(self.level)[0]
+            or self.tile_size[1] > slide.level_dimensions(self.level)[1]
+        ):
```

Comment:
this expression can be simplified in this way i guess:
```python
if [i for i, j in zip(a, b) if i > j]:
    raise ....
``` 
where `a == self.tile_size` and `b ==slide.level_dimensions(self.level)`

so

```python
if [i for i, j in zip(self.tile_size, slide.level_dimensions(self.level)) if i > j]:
    raise ...
```

Reply:
I see where the benefit could be if we had to check more dimensions, but IMHO, since we have only 2 dimensions to check, this simplification is making it harder to understand

Reply:
ok